cmd/zfs/zfs_main.c

```diff
@@ -2529,7 +2529,8 @@ userspace_cb(void *arg, const char *domain, uid_t rid, uint64_t space)
 		namelen = strlen(name);
 	}
 	nameidx = us_field_index("name");
-	if (namelen > cb->cb_width[nameidx])
+	if (nameidx != -1 &&
```

Comment:
I know that us_field_index can only return -1 if it's not successful, but I feel the check should be more generic and cover all negative numbers. So, I feel this should be `nameidx >= 0`